src/main/kotlin/car/domain/CarMoveHistoryCollection.kt

```diff
@@ -0,0 +1,5 @@
+package car.domain
+
+class CarMoveHistoryCollection(moveHistories: List<CarMoveHistory>) {
+    val moveHistories: List<CarMoveHistory> = moveHistories
+}
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
class CarMoveHistoryCollection(val moveHistories: List<CarMoveHistory>)
```